### DISCUSSION TITLE
It should be ind_healthcare_ everywhere not ind_health_care

### DISCUSSION THREAD

**Discussion by lanawwas** [impact-initiatives-hppu/humind#412] [General]
In the [add_com_health.R#L62, 61 and 51](https://github.com/impact-initiatives-hppu/humind/blob/02a748112bd64760e1ae69d4bbfd7df565e44a5a/R/add_comp_health.R#L51) > We need to change the current code from "ind_health_care_" to "ind_healthcare_" 

The changes will be updated in the unit-test branch @gnoblet

**Comment by gnoblet** [comment]
Thanks for the change!

**Comment by lanawwas** [comment]
Solved in [59efc98](https://github.com/impact-initiatives-hppu/humind/commit/59efc98f3e1b9e1ab45d88ff876474b1ed2da150)
